courses/machine_learning/deepdive2/structured/labs/5a_train_keras_ai_platform_babyweight.ipynb

```diff
@@ -0,0 +1,749 @@
+{
```

Comment:
<p>see comment in solution below</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/674/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>Done</p>

Reply:
<p>yeah, I believe I understand what export is supposed to do but when I keep it this way the PROJECT variable is not recognized in later cells. </p><p>Maybe I have something set up incorrectly? Could you double check this works for you as is?</p>

Reply:
<p>I added it back in.</p>